src/Kafka/KafkaMessageBrowser/KafkaMessageBrowser.tsx

```diff
@@ -266,6 +266,11 @@ export const KafkaMessageBrowserConnected: VoidFunctionComponent<
                     </ToolbarItem>
                     <ToolbarItem>
                       <RefreshButton
+                        tooltip={
```

Comment:
I wonder if this shouldn't be the default behavior of the tooltip. So users of the component can simply use it without having to provide the common strings, while users that require some variant of the text can still provide it.